.github/workflows/cre-local-env-tests.yaml

```diff
@@ -55,6 +56,8 @@ jobs:
           filters: |
             ci_changes:
               - '.github/workflows/cre-local-env-tests.yaml'
+            cre_env_changes:
+              - 'core/scripts/cre/environment/**/*'
```

Comment:
why do we need it? all `system-tests` already start the environment before they run, this is superfluous